libs/langchain/langchain/chat_models/databricks.py

```diff
@@ -0,0 +1,47 @@
+import logging
+from typing import List, Optional
+
+from langchain_core.pydantic_v1 import BaseModel, Extra
+
+from langchain.chat_models.mlflow import ChatMlflow
+
+logger = logging.getLogger(__name__)
+
+
+# Ignoring type because below is valid pydantic code
+# Unexpected keyword argument "extra" for "__init_subclass__" of "object"  [call-arg]
+class ChatParams(BaseModel, extra=Extra.allow):  # type: ignore[call-arg]
+    """Parameters for the `MLflow` LLM."""
+
+    temperature: float = 0.0
+    n: int = 1
+    stop: Optional[List[str]] = None
+    max_tokens: Optional[int] = None
+
+
+class ChatDatabricks(ChatMlflow):
+    """`Databricks` chat models API.
+
+    To use, you should have the ``mlflow`` python package installed.
+    For more information, see https://mlflow.org/docs/latest/llms/deployments/databricks.html.
```

Comment:
Why do we only need `mlflow` package for `ChatDatabricks` but need `mlflow[genai]` package for `ChatMlflow`? I thought the two should have the same dependencies.

Reply:
Good question. `ChatMlflow` uses `MlflowDeploymentsClient`. `MlflowDeploymentsClient` requires pydantic (optional dependency).

Reply:
Not sure if I understand. Isn't `ChatDatabricks` basically a clone of `ChatMlflow`? Why do they have different dependencies?

Reply:
- `ChatDatabricks` uses `DatabricksDeploymentsClient` (doesn't require pydantic)
- `ChatMlflow` uses `MlflowDeploymentsClient` (requires pydantic)

Reply:
I see. You are saying that when we do `get_deploy_client("databricks")`, it does not require mlflow[genai] but when we do `get_deploy_client("http://127.0.0.1:5000")`, it requires mlflow[genai]. Is that correct?

Since `ChatDatabricks` and `ChatMlflow` are basically the same, should we add some validation rules to `target_uri` in `ChatDatabricks` so that the user can NOT do:
```
chat = ChatDatabricks(target_uri="http://127.0.0.1:5000", endpoint=name)
```
I am thinking a rule that require "databricks" as the prefix of the target uri?

Reply:
Yes, agree. Should require that target is `databricks` or is a URI with scheme `databricks`

Reply:
I'll add validation.